src/styles/globals.scss

```diff
@@ -334,6 +334,12 @@
     --Link-icon-size: pxToRem(14px);
     font-size: pxToRem(14px);
     font-weight: var(--sds-fw-semi-bold);
+
+    // to fix Safari only styling bug
+    .Link__icon {
+      width: pxToRem(14px);
+      width: pxToRem(14px);
```

Comment:
Should one of them be `height`?

Reply:
@quietbits i fixed it in [this PR ](https://github.com/stellar/laboratory/pull/930/files?w=1#diff-0c4f41e0e71564707ab458bc5d11b7ce941e19d10833a6c46c2995b8341de145R350)